src/Build/Evaluation/Evaluator.cs

```diff
@@ -645,33 +619,23 @@ private void Evaluate(ILoggingService loggingService, BuildEventContext buildEve
 
                 ErrorUtilities.VerifyThrow(_data.EvaluationId != BuildEventContext.InvalidEvaluationId, "Evaluation should produce an evaluation ID");
 
-#if MSBUILDENABLEVSPROFILING
-        string endPass0 = String.Format(CultureInfo.CurrentCulture, "Evaluate Project {0} - End Pass 0 (Initial properties)", projectFile);
-        DataCollection.CommentMarkProfile(8816, endPass0);
-#endif
-
+                EvaluateEventSource.Log.EvaluatePhase(projectFile, 0);
```

Comment:
Why are these not start/stop pairs per phase?

Reply:
These are intermediate points within Evaluate. It's a relatively long function, so it's helpful to be able to find more precisely where the slow points are, but it doesn't "start" or "stop" at these points, only at the very beginning and end respectively.

Reply:
Also, there were CodeMarkers there originally, if I remember correctly.